opensrp-chw/src/ba/java/org/smartregister/chw/interactor/PncHomeVisitInteractorFlv.java

```diff
@@ -780,8 +790,16 @@ public String evaluateSubTitle() {
             if (date == null) {
                 return null;
             }
-
-            return MessageFormat.format("{0} : {1}", context.getString(R.string.date), new SimpleDateFormat("dd MMM yyyy", Locale.getDefault()).format(date));
+            if ("No".equals(pnc_visit)) {
+                return "Visit not done";
+            } else {
+                if (visit_num == 1) {
+                    return MessageFormat.format(" {0}: {1} \n {2}: {3} \n {4}: {5}",
```

Comment:
Use strings.xml or any other translated text for any displayed text

Reply:
Cool. Added and referenced the text in strings.xml